package.json

```diff
@@ -31,14 +31,17 @@
     "@types/jquery": "^3.3.22",
     "chart.js": "^2.7.3",
     "chartjs-plugin-datalabels": "^0.4.0",
+    "core-js": "3",
     "jquery": "^3.4.0",
     "jquery-modal": "^0.9.1",
+    "jquery.mousewheel": "^3.1.9",
```

Comment:
Are you sure this is the right version and that `jquery-mousewheel` isn't needed? This one seems to be abandoned.

Reply:
Yeah, I first went with the `-` variant, which did not work as intended.